modules/apps/headless/headless-admin-user/headless-admin-user-impl/rest-openapi.yaml

```diff
@@ -123,6 +142,12 @@ components:
                     format: int64
                     readOnly: true
                     type: integer
+                keywords:
```

Comment:
`keyword` is a name already used for the same purpose (so assetTag) in other places?